### ISSUE TITLE
GHSL-2021-115

### ISSUE THREAD

**Issue by kevinbackhouse** [arskom/spyne#669]
Hello,

The [GitHub Security Lab team](https://securitylab.github.com/) has found a potential vulnerability in your project. Please create a [Security Advisory](https://docs.github.com/en/free-pro-team@latest/github/managing-security-vulnerabilities/creating-a-security-advisory) and invite me in to further disclose and discuss the vulnerability details and potential fix. Alternatively, please add a [Security Policy](https://docs.github.com/en/free-pro-team@latest/github/managing-security-vulnerabilities/adding-a-security-policy-to-your-repository) containing a security email address to send the details to.

If you prefer to contact us by email, please reach out to securitylab@github.com with reference to GHSL-2021-115.

Thank you,  
Kevin Backhouse
GitHub Security Lab

**Comment by plq** [comment]
yes, I don't want to go into too much detail but it's not the most serious kind like rce, information disclosure etc.

I'll look into this as time permits.
